genienlp/tasks/almond/__init__.py

```diff
@@ -132,6 +133,10 @@ def get_splits(self, root, **kwargs):
         return AlmondDataset.return_splits(path=os.path.join(root, 'almond'), make_example=self._make_example, **kwargs)
 
     def postprocess_answer(self, answer):
+    
+        if self._almond_detokenize_sentence:
+            answer = tokenize_cjk_chars(answer)
```

Comment:
Why is this needed? We should add a comment explaining how CJK are handled by genie and genienlp, and why.

Reply:
It makes genienlp transparent to the changes in cjk chars spacing. 
Sure, I will add a comment.